samples/client/src/i18n.utils.js

```diff
@@ -0,0 +1,78 @@
+import * as vipCore from '@vip/vip-core-sdk';
+import { ENGLISH } from './source.l10n';
+
+const langCookieName = 'locale';
+
+// The attribute in the DOM elements that holds the key for the translation.
+const localizeAttribute = 'l10n';
+
+// Should be executed in the html file that needs to be localized. 
+export function localize() {
+    const lang = detectLanguage();
+
+    // Provide a conventient way to turn on pseudo translation.
+    const shouldUsePseudoTranslations = () => {
+        if (typeof window === 'undefined' || typeof window.localStorage === 'undefined') {
+            return false;
+        }
+        return window.localStorage.getItem('enable_localization_pseudo') === 'true';
+    };
+
+    let i18nClient = vipCore.i18nClient.init(
+        {
+            productID: 'CoreSDKClient',
+            version: '1.0.0',
+            component: 'ui',
+            host: 'http://localhost:8091',
+            language: lang,
+            sourceBundle: ENGLISH,
+            i18nAssets: 'src/sources/',
```

Comment:
core SDK doesn't support 'i18nAssets, we may need to remove it.

Reply:
It actually supports